iiif_prezi3/base.py

```diff
@@ -13,7 +13,10 @@ def __getattribute__(self, prop):
         val = super(Base, self).__getattribute__(prop)
         # __root__ is a custom pydantic thing
         if hasattr(val, '__root__'):
-            return str(val.__root__)
+            if type(val.__root__) in [dict, list]:
```

Comment:
I think this might still be too broad (allows conversion of things like seeAlso to a string) but lets go with it until we hit issues.

Reply:
Agreed both that it's not quite right, and that we can merge and fix other cases as they arise